File: Python/Ex_36.py
values = []
values2 = []

for c in range(0, 20):
    v = float(input("Enter with a value: "))
    values.append(v)

multiplier = float(input('Enter with a multiplier: '))

for i in range(0, 20):
    multiple = values[i] * multiplier
    values2.append(multiple)

for value in values:
    print(value)
    
for value in values2:
    print(value)
 

File: c/Ex_36.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    int values[20], values2[20];
    float multiplier, multiple;

    for (int i = 0; i < 20; i++)
    {
        printf("Enter with a value: ");
        scanf("%i", &values[i]);
    }

    printf("Enter with a multiplier: ");
    scanf("%f", &multiplier);

    for (int i = 0; i < 20; i++)
    {
        multiple = values[i] * multiplier;
        values2[i] = multiple;
    }

    for (int i = 0; i < 20; i++)
    {
        printf(values[i]);
    }

    for (int i = 0; i < 20; i++)
    {
        printf(values2[i]);
    }
    
    return 0;
}